learn/structuring-ballerina-code.md

```diff
@@ -396,14 +376,10 @@ ballerina push <module-name>
 ```
 ### Configuring Ballerina Central Access
 
-Ballerina Central requires an account in order to push modules. Your account is represented by a CLI token that is 
-installed into your local Ballerina configuration file, i.e., `~/.ballerina/Settings.toml`. The CLI token is 
+Ballerina Central requires an account in order to push modules. Your account is represented by a CLI token that is installed into your local Ballerina configuration file, i.e., `~/.ballerina/Settings.toml`. The CLI token is 
 automatically installed into this file the first time you perform a `ballerina push` as Ballerina redirects to an 
-OAuth authorization screen, configures your account, and then copies your CLI token from Ballerina Central into your 
-local CLI configuration. To get your token, register on Ballerina Central and visit the 
+OAuth authorization screen, configures your account, and then copies your CLI token from Ballerina Central into your local CLI configuration. To get your token, register on Ballerina Central and visit the 
 [Ballerina Central Dashboard](https://central.ballerina.io/dashboard).
 
 Every push of the same module into Ballerina Central REQUIRES a new version even for minor text updates. 
-This policy is enforced to ensure that projects, which make use of dependencies cannot experience accidental behavior drift across 
-two versions of the same module given the same version. Essentially, there is no way to "update" a module for a specific 
-version in Ballerina Central.
+This policy is enforced to ensure that projects, which make use of dependencies cannot experience accidental behavior drift across two versions of the same module given the same version. Essentially, there is no way to "update" a module for a specific version in Ballerina Central.
```

Comment:
We don't need this comma.

Reply:
We do because the 'which' is only for 'makes use of dependencies'. It's an additional piece of information. Even when taken out the sentence makes sense.

Reply:
+1